Bacpac/Restore-BcDatabaseFromArtifacts.ps1

```diff
@@ -140,7 +143,7 @@ try {
             }
 
             Write-Host "Restoring database $dbName into $destinationPath"
-            New-NAVDatabase -DatabaseServer $databaseServer -DatabaseInstance $databaseInstance -DatabaseName $dbName -FilePath $bakFile -DestinationPath $destinationPath | Out-Null
+            New-NAVDatabase -DatabaseServer $databaseServer -DatabaseInstance $databaseInstance -DatabaseName $dbName -FilePath $bakFile -DestinationPath $destinationPath -timeout $sqlTimeout | Out-Null
```

Comment:
This code is running in a background job (Start-Job in line 60) and argumentlist in line 206.
You have to transfer the sqlTimeout value to that job - else it is not defined.

Reply:
Absolutely, I had it hardcoded and forgot to add it to the arguments. Updated the argument lists